pkgs/top-level/all-packages.nix

```diff
@@ -7363,6 +7363,9 @@ let
 
   libspectre = callPackage ../development/libraries/libspectre { };
 
+  libspnav = callPackage ../development/libraries/libspnav { };
+  spacenavd = callPackage ../misc/drivers/spacenavd { };
```

Comment:
`spacenavd` should be moved to the misc section (look for `### MISC`) in `all-packages.nix`, that should be around line 15300.